infra/testing/sw-env-mocks/Headers.js

```diff
@@ -17,8 +17,12 @@
 // Stub missing/broken Headers API methods in `service-worker-mock`.
 // https://fetch.spec.whatwg.org/#headers-class
 class Headers {
-  constructor(obj = {}) {
-    this.obj = Object.assign({}, obj);
+  constructor(init = {}) {
```

Comment:
We've diverged from `service-worker-mock` over the years, but this change is roughly https://github.com/pinterest/service-workers/blob/c0eda82a98aa46d9a6fb44a32906fa0c2a4f2b22/packages/service-worker-mock/models/Headers.js#L4-L12